tidb-cluster-on-centos/azuredeploy.parameters.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+        "value": "tidb"
+    },
+    "adminPassword": {
+        "value": "GEN-PASSWORD"
+    },
+    "storageAccountNamePrefix": {
+      "value": "GEN-UNIQUE-5"
+    },
+    "virtualNetworkName": {
```

Comment:
for any values that are not "GEN" values, just use defaultValues on the parameters in the template